src/main/java/world/bentobox/bentobox/api/commands/admin/DefaultAdminCommand.java

```diff
@@ -92,7 +92,7 @@ public void setup() {
 
     /**
      * Defines what will be executed when this command is run.
-     * @see world.bentobox.bentobox.api.commands.BentoBoxCommand#execute(User, String, List<String>)
+     * @see world.bentobox.bentobox.api.commands.BentoBoxCommand#execute(User, String, List(String))
```

Comment:
What does that change?

Reply:
Chevrons are illegal characters in Javadocs. It's like HTML.

Reply:
Oh, okay... Good to know.